nav-app/src/app/datatable/data-table.component.html

```diff
@@ -496,6 +496,113 @@
+                     <img [src]="viewModel.isCurrentlyEditing() && viewModel.metadataMatch ? 'assets/icons/ic_metadata_black_24px.svg' : 'assets/icons/ic_metadata_gray_24px.svg'" alt="metadata" />
+                 </div>
+                 <!-- link input -->
+                 <div class="dropdown-container inputfield left list-input" #dropdown [class.left]="checkalign(dropdown)"
+                    *ngIf="currentDropdown === 'metadata'">
+                    <div class="list-container">
+                        <div *ngFor="let metadata of metadataEditField; let i = index">
+                            <div class="item-container" *ngIf="!metadata.divider">
+                                <mat-form-field>
+                                    <textarea matInput
+                                            matTextareaAutosize
+                                            placeholder="name"
+                                            [(ngModel)]="metadataEditField[i].name"
+                                            (input)="updateMetadata()">
+                                    </textarea>
+                                </mat-form-field>
+                                <mat-form-field>
+                                    <textarea matInput
+                                            matTextareaAutosize
+                                            placeholder="value"
+                                            [(ngModel)]="metadataEditField[i].value"
+                                            (input)="updateMetadata()">
+                                    </textarea>
+                                </mat-form-field>
+                                <button class="button" (click)="removeMetadata(i)">remove</button>
+                            </div>
+                            
+                            <div class="divider" (click)="onClickDivider(i, !metadata.divider, 'metadata')" *ngIf="canAddDivider(i, 'metadata') || metadata.divider"
+                                matTooltipPosition="below" [matTooltip]="metadata.divider ? 'remove divider' : 'add a divider'">
+                                <div [ngClass]="{'opt-divider': canAddDivider(i, 'metadata'), 'set-divider': metadata.divider}">
+                                    <mat-divider></mat-divider>
+                                    <button mat-icon-button class="icon">{{metadata.divider ? '-': '+'}}</button>
+                                    <mat-divider></mat-divider>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="add-more">
+                        <button class="button" (click)="addMetadata()">add {{metadataEditField.length ? 'more' : ''}} metadata</button>
+                    </div>
+                </div>
+             </div>
```

Comment:
- Again, should be refactored into its own component.
- Behavior here is similar to the link input -- is it possible to consolidate them into a single component and pass in config to determine the fields to read/write?